docs/install-1.6.md

```diff
@@ -0,0 +1,165 @@
+wide variety of customizations which are detailed in that directory's
+[README.md](https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/README.md).
+
+## The Service Catalog Data Store
+
+We'll be interacting with a variety of resources in the following steps. The
+service catalog API server needs to store all of these resources in a data
+store. The data store implementation in the API server is pluggable, and we
+currently support the following implementations:
+
+1. Etcd 3
+2. Third Party Resources (also, known as TPRs) - this is an _alpha_ feature 
+right now. It has known issues and may be removed at any time
+
+The first implementation requires that the API server has access to an Etcd 3 cluster, and the
+second only requires access to the Kubernetes API to store TPRs.
+
+Even if you store data in TPRs, you should still access data via the service catalog API. It is 
+possible to access data via the TPRs directly, but we don't recommend it.
+
+## Install
+
+To install the service catalog system with Etcd 3 as the backing data store:
+
+```console
+helm install charts/catalog --name catalog --namespace catalog
+```
+
+To install the service catalog system with TPRs as the backing data store:
+
+```console
+helm install charts/catalog --name catalog --namespace catalog --set apiserver.storage.type=tpr,apiserver.storage.tpr.globalNamespace=catalog
+```
+
+Regardless of which data store implementation you choose, the remainder of the steps in this
+walkthrough will stay the same.
+
+## API Server Authentication and Authorization
+
+Authentication and authorization are disabled in the Helm chart by default. To enable them, 
+set the `apiserver.auth.enabled` option on the Helm chart:
+
+```console
+helm install charts/catalog --name catalog --namespace catalog --set apiserver.auth.enabled=true
+```
+
+For more information about certificate setup, see the [documentation on
+authentication and authorization](./auth.md).
+
+
+## Do Overs
+
+If you make a mistake somewhere along the way in this walk-through and want to start over,
+check out the "Cleaning Up" section below. Follow those instructions before you start over.
+
+## Step 3 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address that points to the `Service` that sits in front of the API server pod(s).
+If you installed the catalog with one of the `helm install` commands above, then this service 
+will be called `catalog-catalog-apiserver`, and be in the `catalog` namespace. 
+
+### Notes on Getting the IP Address
+
+How you get this IP address is highly dependent on your Kubernetes installation method. Regardless
+of how you do it, do not use the Cluster IP of the `Service`. The `Service` is created as a
+`NodePort` in this walkthrough, so you'll likely need to use the IP address of the node or one of
```

Comment:
"you will need to use the address of one of the nodes in your cluster." We know how NodePorts work, we don't need to hedge our bets.

not sure how to say name or ip, but if you want to throw IP in there as "IP address" go ahead.

Reply:
done